cost-analyzer/values.yaml

```diff
@@ -166,13 +166,12 @@ prometheus:
               record: kubecost_cluster_memory_working_set_bytes
 
 networkCosts:
-  enabled: true
+  enabled: false
   image: gcr.io/guestbook-227502/kubecost-network-costs:v26
   resources:
     #requests:
-    #  cpu: "200m"
-    #  memory: "200Mi"
-
+    #  cpu: "50m"
```

Comment:
I think we should actually ship with some default requests/limits enabled given it's a daemonset. Otherwise, a memory leak or something could cause widespread problems. This can come later this week after we run scale tests.